src/main/js/field/input/DynamicSelect.js

```diff
@@ -4,19 +4,20 @@ import Select from 'react-select';
 import LabeledField from 'field/LabeledField';
 
 function noOptionsMessage() {
-    return null;
+    return 'No options available...';
```

Comment:
This function can be removed and the noOptions prop can be passed {() => 'No options available...'} instead